keeper-gradle-plugin/build.gradle.kts

```diff
@@ -48,6 +48,10 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach
     }
 }
 
+sourceSets {
+    findByName("test")?.resources?.srcDirs("$buildDir/pluginUnderTestMetadata")
```

Comment:
nit: can we use `getByName` and fail fast if this isn't working?

Reply:
daf340b